backend/src/bigQuery.ts

```diff
@@ -1,21 +1,30 @@
 import { BigQuery } from "@google-cloud/bigquery";
 
-const options = {
-  keyFilename: process.env.DEV
-    ? "../gcloud/service-account.json"
-    : "../../../gcloud/service-account.json",
-  projectId: "hubble-261722",
-};
+let options;
+if (process.env.DEV) {
+  options = {
+    keyFilename: "gcloud/service-account.json",
+    projectId: process.env.BQ_PROJECT_ID,
+  };
+} else {
+  options = {
+    keyFilename: "../gcloud/service-account.json",
```

Comment:
this may not be the right location of the `service-account.json` in production before it was `keyFilename: "../../../gcloud/service-account.json"`, but we can check in the preview environment before merging

Reply:
oh it doesn't look like previews show up on forks, let me create a branch on the dashboard repo with these changes to check

Reply:
The tests have been running for an awfully long time, the only noticeable difference i could notice is that our linter is changing the health checks on redis, i'll push a modification to circumvent that. Current workflow can be stopped though, it seems to be on an infinite loop.

Reply:
I pushed the new commit, it adds our .github folder to the prettierignore list. It's a long shot as to what may be causing the loop, but github actions are notoriously iffy about formatting so it may be it. I'll keep an eye on the next workflow to get the logs.

Reply:
also let's change the gcloud prod file location back to `keyFilename: "../../../gcloud/service-account.json",`. Doesn't look like the preview environment pod can find the file currently and this was what it was set to previously that I'm fairly certain was working

Reply:
Done! ammended the previous commit.

Reply:
I got the logs this time, this recent test also failed and will run indefinitely. I'll investigate, meanwhile the last workflow can be cancelled. Apparently the packages are not being installed.

Reply:
Pushing a new commit with the following changes:
- Reintroduced a forced exit after all tests run. Dangling redis instances were causing the forever loop.
- Added a new step to our github action to install the dependencies of our commons scripts. Since lumens makes use of src/commons/lumens.js and that requires a different set of packages, they have to be installed separately.

Hopefully the tests will run without issue after these changes.

Reply:
nice catches 👍

Reply:
nice catches 👍

Reply:
whats the merge policy @acharb ? can i merge once it's approved or is there another proccess?

Reply:
yup after approved you can merge 👍